ROADMAP.md

```diff
@@ -0,0 +1,14 @@
+This is a list of what the Purple Bubble project has done, and what still needs to be done.
+
+> **Contributors:** Feel free to help out with any of these tasks! The Purple Bubble team & community communicates via Slack. You can join the slack at https://join.slack.com/t/thepurplebubble/signup. If you can't join or would prefer not to, reach out to team@purplebubble.org or open a github issue if you have questions or need help.
```

Comment:
Let's add PR templates for helping with common tasks

Reply:
shouldn't those go into a template repo? I can make a PR there